### ISSUE TITLE
Medborg Roller bed rack should change sprites when empty

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#14997]
(WEB REPORT BY: sonixapache REMOTE: 172.93.109.202:7777)
>Revision (Should be above if you're viewing this from ingame!)

d4360fb2c3ee6931875a3c97e51a8e77a06c3834 

>General description of the issue

you can't easily tell if you've got a bed on you or not
t. mouse (385)

**Comment by jknpj** [comment]
WYSI

**Comment by Bilgecrank** [comment]
![borgbed](https://user-images.githubusercontent.com/8432868/27990921-0e92f074-6434-11e7-96d8-f7b2c2628293.png)

I decided to go a step further with this one since I too have played med-borg and misplaced my bed more than once. I made a custom bed for the MedBorg specifically, so that their bed stands out from the other beds for Med-Bay's use. I've prepared a [DMI](https://www.dropbox.com/s/iqf408b0bhwwca7/borgbed.dmi?dl=0) with the sprites within, but if that's too much, we could just use the DEPLOYED sprite instead.
